python/ray/worker.py

```diff
@@ -1682,16 +1682,27 @@ def kill(actor):
 
 
 def cancel(object_id, force=False):
-    """Kill a task forcefully.
+    """Cancels a locally-submitted task according to the following pseudocode.
```

Comment:
according to the following logic? procedure? conditions?

Reply:
Resovled